tests/test_mutations.py

```diff
@@ -1087,6 +1087,60 @@ def test_arbitrary_model(self):
         self.verify_mutation_rates(model)
 
 
+class TestSlimMutationGenerator(unittest.TestCase):
+    """
+    Tests for the SLiM mutation generator.
+    """
+
+    def validate_slim_mutations(self, ts, increasing=True):
+        # slim alleles should be unique comma-separated lists of integers
+        # and ancestral states the empty string
+        # if `increasing` then each one should have one more integer than the one before
+        alleles = {}
+        for s in ts.sites():
+            self.assertEqual(s.ancestral_state, "")
+            for m in s.mutations:
+                a = tuple(m.derived_state.split(","))
```

Comment:
This is different to the C generator now though, isn't it? There we're using little-endian 8 byte values and here we're using comma delimited text. Which one does SLiM do?

Reply:
Whoops, yes. SLiM does the [concatenated bytes thing](https://github.com/MesserLab/SLiM/blob/1c85d00a0417416431ca69f2cb68a1bcc134ec3d/core/slim_sim.cpp#L5137), not text. I was waiting to compare these until we've hooked the slim generator up to python.